codebuild/bin/install_default_dependencies.sh

```diff
@@ -16,6 +16,7 @@
 set -ex
 source codebuild/bin/s2n_setup_env.sh
 
+OPENSSL_1_1_1_DEFAULT_VERSION="1.1.1g"
```

Comment:
Should we bump this?

Reply:
Yes, although after a rebase, it'll show up in the `install_openssl_1_1_1.sh`